README.md

```diff
@@ -22,7 +22,8 @@ Built with the [Meltano Singer SDK](https://sdk.meltano.com).
 | client_secrets      | False    | None    | Google Analytics Client Secrets Dictionary |
 | key_file_location   | False    | None    | File Path to Google Analytics Client Secrets |
 | oauth_credentials   | False    | None    | Google Analytics OAuth Credentials |
-| reports             | False    | None    | Google Analytics Reports Definition |
+| reports             | False    | None    | File Path to Google Analytics Reports Definition |
```

Comment:
This should match the descriptions in tap.py, I left some suggestions too so if you accept those please update this to match so its consistent. You can also run `tap-google-analytics --about --format=markdown` to get this top part of the README auto generated ✨